politeiawww/cmd/politeiawwwcli/commands/edituser.go

```diff
@@ -4,6 +4,20 @@ import (
 	"github.com/decred/politeia/politeiawww/api/v1"
 )
 
+// Help message displayed for the command 'politeiawwwcli help edituser'
+var EditUserCmdHelpMsg = `edituser "userid" "action" "reason"
+
+Edit the details for the given user id (admin).
+
+Arguments:
+1. emailnotifications       (uint64, optional)   Whether to notify via emails
```

Comment:
This description should  be something like:

`Email notifications setting.  Uses bit flags.`

Reply:
To keep within 80 char width, shortened to:

```
Arguments:
1. emailnotifications (uint64, optional)  Email notification setting (bit flag)
```